docs/tutorial/part-zero/index.md

```diff
@@ -211,7 +211,7 @@ Now you are ready to use the Gatsby CLI tool to create your first Gatsby site. U
 2. Create a new site from a starter:
 
 ```shell
-gatsby new hello-world https://github.com/gatsbyjs/gatsby-starter-hello-world
+gatsby new hello-world https://github.com/gatsbyjs/gatsby/tree/master/starters/hello-world
```

Comment:
Can you elaborate on this change? It seems unnecessary as I just tested the old version (`gatsby new hello-world https://github.com/gatsbyjs/gatsby-starter-hello-world`) and that works fine. If nothing else, please undo this line.

Reply:
The reason for the URL update is to point to the current repo. While the old link still works, it points to a repo where I received a message saying it would no longer be maintained and to look at the newest documentation on the new site. I imagine all starter files need to be updated but I was only working with the hello-world one.

Reply:
Great point. Thank you!

Reply:
the https://github.com/gatsbyjs/gatsby-starter-hello-world should be a mirror of https://github.com/gatsbyjs/gatsby/tree/master/starters/hello-world - if they are not in sync then this should be checked

Reply:
and i got this error:

```shell
$ gatsby new hello-world https://github.com/gatsbyjs/gatsby/tree/master/starters/hello-world

 ERROR 

starter https://github.com/gatsbyjs/gatsby/tree/master/starters/hello-world doesn't exist



  Error: starter https://github.com/gatsbyjs/gatsby/tree/master/starters/hello-world doesn't exist
  
  - init-starter.js:157 copy
    [lib]/[gatsby-cli]/lib/init-starter.js:157:11
  
  - init-starter.js:345 async initStarter
    [lib]/[gatsby-cli]/lib/init-starter.js:345:58
  
  - create-cli.js:400 
    [lib]/[gatsby-cli]/lib/create-cli.js:400:7

```

Reply:
is see there are already a PR for this:

- #26504 `In Tutorial Docs Step 0, wrong git clone URL is used`

Reply:
Touching on this - the automation that we used to sync read-only repositories was broken since Aug 12th. I fixed the sync automation yesterday, so this change should be reverted - and the https://github.com/gatsbyjs/gatsby/pull/26528 seems to be doing that